google/resource_compute_global_address_test.go

```diff
@@ -29,6 +31,29 @@ func TestAccComputeGlobalAddress_basic(t *testing.T) {
 	})
 }
 
+func TestAccComputeGlobalAddress_ipv6(t *testing.T) {
+	var addr computeBeta.Address
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeGlobalAddressDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccComputeGlobalAddress_ipv6,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeBetaGlobalAddressExists(
```

Comment:
I'm not sure I agree that we need to validate that the API isn't lying to us when it says an IP address is at some version; especially when we would need to use a regex or vendor a new package to verify it. 

Added a test for IPV4 on `_basic`, and moved checks to a TestCheckFunc.